crates/apollo-compiler/Cargo.toml

```diff
@@ -17,7 +17,7 @@ categories = [
 edition = "2021"
 
 [dependencies]
-apollo-parser = { path = "../apollo-parser", version = "0.5.0" }
+apollo-parser = { path = "../apollo-parser", version = "0.5.3" }
```

Comment:
should i bump this to `0.5.3`? or just leave it at `0.5.0`; the latest is currently within the semver range, so probably `0.5.3` is not necessary eh?